service/src/main/java/bio/terra/pipelines/notifications/TeaspoonsJobSucceededNotification.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.pipelines.notifications;
+
+import lombok.Getter;
+
+@Getter
+public class TeaspoonsJobSucceededNotification extends BaseTeaspoonsJobNotification {
+  private static final String NOTIFICATION_TYPE = "TeaspoonsJobSucceededNotification";
+  public final String notificationType;
+  public final String quotaConsumedByJob;
+
+  public TeaspoonsJobSucceededNotification(
+      BaseTeaspoonsJobNotification baseTeaspoonsJobNotification, String quotaConsumedByJob) {
```

Comment:
this is a bit of an awkward pattern for a sub class to have its parent class as constructor argument.  prob worth talkign about to see if there is a way to not do this